ajax/libs/smooth-scrollbar/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "smooth-scrollbar",
+  "filename": "smooth-scrollbar.js",
+  "version": "6.0.1",
+  "description": "Customize scrollbar in modern browsers with smooth scrolling experience.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/idiotWu/smooth-scrollbar.git"
+  },
+  "keywords": [
+    "scrollbar",
+    "customize",
+    "acceleration",
+    "performance"
+  ],
+  "author": "Dolphin Wood <dolphin.w.e@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://idiotwu.github.io/smooth-scrollbar/",
+  "npmName": "smooth-scrollbar",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Hmm...actually I'm thinking that if is necessary to publish a unminified version of compiled source. Since the project is written in es6 and pre-compiled with babel, it would be hard to debug without souce maps.